developer-extension/src/panel/components/alert.tsx

```diff
@@ -15,7 +15,7 @@ export function Alert({
 }) {
   const color = level === 'warning' ? ('orange' as const) : ('red' as const)
   return (
-    <Center mt="xl">
+    <Center mt="xl" className={'dd-privacy-allow'}>
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
    <Center mt="xl" className="dd-privacy-allow">
```